slackbot/plugins/_Xoldjira

```diff
@@ -0,0 +1,87 @@
+    @botcmd(split_args_with=';')
```

Comment:
Need some cleanup in here.  Requisite docstring and break the code into smaller chunks.